adder-viz/src/transcoder/ui.rs

```diff
@@ -153,8 +153,7 @@ impl TranscoderState {
         ui.horizontal(|ui| {
             if ui.button("Open file").clicked() {
                 if let Some(path) = rfd::FileDialog::new()
-                    .add_filter("framed video", &["mp4"])
-                    .add_filter("DVS/DAVIS video", &["aedat4"])
+                    .add_filter("video", &["mp4", "aedat4"])
```

Comment:
Keep these separate under the builder pattern, because they are semantically entirely different. This hurts readability.

Reply:
Also keep your PR focused on a single issue. This one is for getting it to build on Windows, so all your changes should only be towards that end.

Reply:
[builder pattern info](https://rust-unofficial.github.io/patterns/patterns/creational/builder.html)

And aedat4 and mp4 are completely different classes of data, so they should be treated separately for readability